pydash-front/src/authenticated_app/overview/AddDashboardDialog.js

```diff
@@ -48,20 +50,38 @@ class AddDashboardDialog extends Component {
             name = this.state.name,
```

Comment:
Please split this function into multiple pieces, similarly as to the registration page one.